testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -358,7 +358,9 @@ def test_rmsf_single_frame(self, universe):
                             err_msg="error: rmsfs should all be zero")
 
     def test_rmsf_old_run(self, universe):
-        rmsfs = rms.RMSF(universe.select_atoms('name CA')).run(start=5, stop=6)
+        with pytest.deprecated_call():
+            rmsfs = rms.RMSF(universe.select_atoms('name CA')).run(start=5,
+                                                                   stop=6)
```

Comment:
Would it be helpful to add a brief comment indicating why this is deprecated (i.e., how this should be done now)?